src/xsdba/utils.py

```diff
@@ -922,6 +966,129 @@ def _skipna_correlation(data):
     ).rename("correlation")
 
 
+def bin_width_estimator(X):
+    """Estimate the bin width of an histogram.
+
+    References
+    ----------
+    :cite:cts:`sdba-robin_2021`
```

Comment:
Has this been added to `references.bib`?

Reply:
I should remove `sdba` from all references